Drivers/Src/CAN.c

```diff
@@ -0,0 +1,68 @@
+#include "CAN.h"
+#include "config.h"
+
+
+/**
+ * @brief   Initializes the CAN system
+ * @param   None
+ * @return  None
+ */
+void CAN_Init(void) {
+    BSP_CAN_Init(CAN_1);
+}
+
+/**
+ * @brief   Transmits data onto the CANbus
+ * @param   CAN bus line
+ * @param   id : CAN id of the message
+ * @param 	payload : the data that will be sent.
+ * @return  0 if data wasn't sent, otherwise it was sent.
+ */
+int CAN_Send(CANId_t id, CANPayload_t payload) {
+
+	switch (id) {
+		case MC_BUS:
+		case VELOCITY:
+		case MC_PHASE_CURRENT:
+		case VOLTAGE_VEC:
+		case CURRENT_VEC:
+		case BACKEMF:
+		case TEMPERATURE:
+		case ODOMETER_AMPHOURS:
+        case MOTOR_DISABLE:

```

Comment:
Don't need to send this MOTOR_DISABLE message, so it can be removed from this function